.github/workflows/pytest.yml

```diff
@@ -43,6 +43,13 @@ jobs:
       - name: pip installs
         run: pip install -e .[test]
 
+      - name: setup matplotlib
```

Comment:
Note that the default matplotlib backend used on Windows is somehow broken (TK library cannot be found for whatever reason). To work around this I forcefully use the Agg backend for this OS.